Documentation/Contributing/development-flow.md

```diff
@@ -242,6 +245,176 @@ Common cases that may need tests:
 * an input is not specified, for each input
 * an input is specified incorrectly, for each input
 * a resource the code relies on doesn't exist, for each dependency
+
+## Linting
+
+The CI will run various linters against your changes to check code, manifests, and docs for syntactic correctness.
```

Comment:
I agree with Blaine, let's have fewer paragraphs, no need for each sentence in a separate paragraph

Reply:
done for this case.

Reply:
Please look at consolidating others as well.

Reply:
done i some places that I spotted.